scripts/cronJobs.js

```diff
@@ -0,0 +1,32 @@
+const cron = require('node-cron');
+const userRecs = require('./simulateUserRecs.js');
+const genreRecs = require('./calculateGenreRecs');
+
+// simultate GET requests for individual user recommendations
+cron.schedule('* * * * *', () => {
```

Comment:
You might be able to add some helper functions here to help with DRY code (e.g. a random number generator function).

Reply:
I added a random number generator function and called the cron schedule 10 times in a for loop instead of writing out the code 10 time.  My code is much cleaner now.